test/config/fixtures/config-schema.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and other contributors where applicable.
+ * Licensed under the BSD 2-Clause License; you may not use this file except in
+ * compliance with the BSD 2-Clause License.
+ */
+
+'use strict'
+
+const {
+  // CAPTURE_ERROR_LOG_STACK_TRACES_MESSAGES,
+  CENTRAL_CONFIG_OPTS,
+  DEFAULTS,
+  DURATION_OPTS,
+  ENV_TABLE,
+  BOOL_OPTS,
+  NUM_OPTS,
+  BYTES_OPTS,
+  MINUS_ONE_EQUAL_INFINITY,
+  ARRAY_OPTS,
+  KEY_VALUE_OPTS
+} = require('../../../lib/config/schema')
+
+// Create schema for tests
+const CONFIG_SCHEMA = Object.keys(DEFAULTS).reduce((acc, key) => {
```

Comment:
Do you intend to define the `CONFIG_SCHEMA` in "./lib/config/..." files eventually?  I'm a little confused why we would have an intermediate schema object just defined in the tests.

Reply:
Indeed that's the intent although it may have some slight changes at the end. The place for `CONFIG_SCHEMA` shall be `./lib/config/schema.js`

I think this tests will give us more confidence to move to the new schema before doing the refactor (in which we will remove this file and use the actual schema)